web-test-runner.config.js

```diff
@@ -0,0 +1,57 @@
+import {chromeLauncher} from '@web/test-runner';
+import {resolve} from 'path';
+import {symlinkSync, existsSync, unlinkSync, lstatSync, rmdirSync} from 'fs';
+
+function findDeployFolder() {
```

Comment:
Please add a way to run locally, using an argument like `--deploy path/to/deploy`

Reply:
There is the environment variable for non-standard installation locations, otherwise this is as local as it gets, CI setup is not part of the PR

Reply:
As long as I can overwrite it per launch it's fine then

Reply:
The goal to me is to be able to run with a debug build easily

Reply:
Yes, that was the idea, basically `DEPLOY_FOLDER=../../something npm run test`, I'll add instructions to the README